cmos.js

```diff
@@ -27,15 +28,15 @@ define([], function () {
 
         function cmosRead(IC32) {
             // b-em comment is: To drive bus, CMOS must be enabled, D must be high, RW must be high.
```

Comment:
If possible, can this comment be updated, and if not, maybe removed? Now we're diverging from what b-em does the comment doesn't match the code beneath (as is so often the case with comments...)

Reply:
Good point, will update.